test/bun.js/child_process.test.ts

```diff
@@ -0,0 +1,71 @@
+import { describe, it, expect } from "bun:test";
+import { ChildProcess, spawn } from "node:child_process";
+
+describe("ChildProcess.spawn()", () => {
```

Comment:
Can you port some of the node tests for child_process to run in Bun?

Reply:
Totally, working on that tonight. These were really just to test the base functionality 👍